crates/ra_lsp_server/src/main_loop/handlers.rs

```diff
@@ -690,7 +690,7 @@ pub fn handle_code_action(
             edit: None,
             command: Some(command),
         };
-        res.push(action);
+        res.push(CodeActionOrCommand::CodeAction(action));
```

Comment:
I wonder is there an `.into()` impl by any chance?

Reply:
There does not appear to be.

Reply:
I've submitted a PR for this to lsp-types.